components/security/introduction.rst

```diff
@@ -21,6 +21,20 @@ You can install the component in 2 different ways:
 
 .. include:: /components/require_autoload.rst.inc
 
+Starting from Symfony 2.4, the Security component is divided into four smaller
+sub-components which can be used separately:
+
+``symfony/security-core``
+    It provides all the common security features, from authentication to
+    authorization and from encoding passwords to loading users.
+``symfony/security-http``
+  It integrates the core sub-component with the HTTP protocol to handle HTTP
+  requests and responses.
```

Comment:
And does the definition list render properly without blank lines separating the entries?